.github/workflows/postman.yml

```diff
@@ -0,0 +1,70 @@
+      - name: Replace hardcoded values with variables
+        run: |
+          sed -i 's/mainnet/"{{network}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/rdx1qspacch6qjqy7awspx304sev3n4em302en25jd87yrh4hp47grr692cm0kv88/"{{sender_address_1}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/rdx1qspcvwwuf8s549zyspz683v4n93g9kzpn6u6a9yalwzt00zghg75lmsftwv29/"{{receiver_address_1}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/rv1qf2x63qx4jdaxj83kkw2yytehvvmu6r2xll5gcp6c9rancmrfsgfwttnczx/"{{validator_address_1}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/ef71a9d6c63444fce6abd2df8fab2755cfb51f6794e578f60d99337193811842/"{{tx_id}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/hpyron_rr1qdukq6q534plnen70k95f80q45cktg0mhmzrzpfyz8jqmgqqqe/"{{token_rri}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/xrd_rr1qy5wfsfh/"{{native_token_rri}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/3046022100cda0fcd31bf976aa65c31c180b4f595a61c866252c52898b952b8fe1d8fdcc33022100999e83036f9529c6250c07423d8b57f252dfba1b843385970bfa69f2c367658e/"{{tx_bytes}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/0d00010776bf65acf2d25e9dcf4c716f5f39f201dccbfa173ad9c7f1c1dbcb8a86776b4d0000000101002100000000000000000000000000000000000000000000000000012901c1cf3900000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7ff8000800000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7f9402004506000402c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5010000000000000000000000000000000000000000000000000000000000000064000c0034303030303663366636313634323037343635373337343639366536373230373437323631366537333631363337343639366636650b00cda0fcd31bf976aa65c31c180b4f595a61c866252c52898b952b8fe1d8fdcc33999e83036f9529c6250c07423d8b57f252dfba1b843385970bfa69f2c367658e/"{{signed_transaction}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/0d00010776bf65acf2d25e9dcf4c716f5f39f201dccbfa173ad9c7f1c1dbcb8a86776b4d0000000101002100000000000000000000000000000000000000000000000000012901c1cf3900000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7ff8000800000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7f9402004506000402c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5010000000000000000000000000000000000000000000000000000000000000064000c003430303030366336663631363432303734363537333734363936653637323037343732363136653733363136333734363936663665/"{{unsigned_transaction}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798/"{{public_key_hex}}"/g' gateway-api-spec.yaml
```

Comment:
Do we also need to fix the signature bytes `3046022100cda0fcd31bf976aa65c31c180b4f595a61c866252c52898b952b8fe1d8fdcc33022100999e83036f9529c6250c07423d8b57f252dfba1b843385970bfa69f2c367658e`?

Reply:
I do fix that, I just named it wrongly (its okay now I think)